composer.json

```diff
@@ -13,6 +13,7 @@
         "doctrine/doctrine-fixtures-bundle"    : "~2.2",
         "doctrine/orm"                         : "~2.4",
         "erusev/parsedown"                     : "~1.5",
+        "ezyang/htmlpurifier"                  : "~4.0",
```

Comment:
please make it ``~4.7`` instead. We don't want older versions (4.6 makes messy things with permissions in its cache folder)